src/sonic-bgpcfgd/tests/data/general/instance.conf/param_all_v4.json

```diff
@@ -1,7 +1,7 @@
 {
     "CONFIG_DB__DEVICE_METADATA": {
         "localhost": {
-            "sub_role": "BackEnd"
+            "sub_role": "FrontEnd"
```

Comment:
I think the sub_role was removed completely from the instance.conf? We don't need it anymore